accounting/bank/feeds/ponto.rst

```diff
@@ -0,0 +1,115 @@
+======================================
+Ponto as Bank Synchronization provider
+======================================
+
+**Ponto** is a service that allows companies and professionals to aggregate their accounts in one
+place and directly see all their transactions within one app.
+
+It is a third-party solution that is continuously expanding the number of bank institutions that can
+be synchronized with Odoo.
+
+.. image:: media/ponto_logo.png
+   :align: center
+   :alt: Logo of the Ponto brand
+
+**Odoo Accounting** can synchronize directly with your bank to get all bank statements imported
+automatically into your database. This allows for easier **bank reconciliation**.
+
+When :doc:`adding a bank account on Odoo <../setup/create_bank_account>`, you can see if your bank
+requires a connection through Ponto by searching for your bank institution, and clicking on it.
+
+.. image:: media/ponto_add_bank.png
+   :align: center
+   :alt: Click on a bank institution to see which third party service is required to synchronize
+         your bank with Odoo Accounting.
+
+.. note::
+   You can find more information about bank synchronization :doc:`on this page
+   <bank_synchronization>`.
+
+Configuration
+=============
+
+Link your bank accounts with Ponto
+----------------------------------
+
+#. Go on `Ponto's website (https://myponto.com) <https://myponto.com>`_.
+#. Create an account if you don’t have one yet.
+#. Once you are logged in, create an *organization*.
+
+   .. image:: media/ponto_organization.png
+      :alt: Fill out the form to add an organization in Ponto
+
+#. | Go to :menuselection:`Accounts --> Live`, and click on **Add account**.
+   | You might have to add your **Billing Information** first.
+#. Select your bank institution and follow the steps on-screen to link your bank account with your
+   Ponto account.
+#. Make sure to add  all the bank accounts you want to synchronize with your Odoo database before
+   moving on to the next steps.
+
+.. _ponto-link-odoo:
+
+Link your Ponto account with your Odoo database
+-----------------------------------------------
+
+Odoo requires you to insert your Ponto's **Client ID** and **Secret ID** to synchronize both
+platforms.
+
+.. image:: media/ponto_link_odoo.png
+   :align: center
+   :alt: Logo of the Ponto brand
+
+#. To generate these IDs, go to :menuselection:`Ponto Dashboard --> Integration --> Live`, click on
+   **Add Integration**, fill out the form, and select which accounts you want to synchronize.
+#. Copy the *Client ID* and *Secret ID* generated and paste them in Odoo.
+#. | Configure the synchronization options:
+   | - *Action*: define if Odoo must create a new *Journal* for this bank account, link to an
+     existing Journal, or create no link.
+   | - *Journal*: select the appropriate *Journal*.
+   | - *Synchronization Frequency*: define how often Odoo should fetch the bank statements.
```

Comment:
Same as above here. If this is something we should do, please let me know 'cause I didn't know (:
Otherwise, I believe when we have bullet points we don't need "|".

Reply:
Here I use the line break because of the limitations of the RST code:
I cannot use nested lists when the main list is a numbered list.
Therefore, I used line breaks to display the following points separately.
The - are not rendered as lists but as text.

Reply:
@AntoineVDV any thoughts about this?
I couldn't do a nested list with the main numbered list. Is my use of line breaks with "-" as text a bad idea?

Reply:
hum..thank you for the explan!

Reply:
RST doesn't like when you mix different formatting.
Insert a blank line before `- *Action* ...` and it will render fine without `|`.

Reply:
OH AND THE NUMBERING KEEPS WORKING if I were to add another #. 
Thank you!
Why am I so excited about this.